### ISSUE TITLE
Readme links broke

### ISSUE THREAD

**Issue by SamSpiri** [jerakia/jerakia#153]
Ex. https://www.craigdunn.org/2017/08/understanding-hierarchical-data-lookups/
is 404 Error

**Comment by SamSpiri** [comment]
And also my bookmarked article https://www.craigdunn.org/2017/08/hierarchical-data-lookups-ansible/ is gone

**Comment by beddari** [comment]
Yeah, sorry about that, there's still some related content at Craigs site if you nav to the top of it. As of now, Jerakia as a project is in a uncertain state as we just don't have enough users or interest to be able to do needed maintenance.

I think if anyone demonstrate they want to work with it @crayfishx would be open to hand out further access to people in this github org ...
